src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -396,5 +391,65 @@ private static void removeTenant(String tenantId) {
       log.error("Tenant clean up failed: " + e.getMessage(), e);
       assert false;
     }
+  }*/
+
+  static private void prepareTenant(String tenantId, boolean loadSample) {
+    prepareTenant(tenantId, null, "mod-circulation-storage-1.0.0", loadSample);
   }
+
+  private static void prepareTenant(String tenantId, String moduleFrom, String moduleTo,
+      boolean loadSample) {
+
+    JsonArray ar = new JsonArray();
+    ar.add(new JsonObject().put("key", "loadReference").put("value", "true"));
+    ar.add(new JsonObject().put("key", "loadSample").put("value", Boolean.toString(loadSample)));
+
+    JsonObject jo = new JsonObject();
+    jo.put("parameters", ar);
+    if (moduleFrom != null) {
+      jo.put("module_from", moduleFrom);
+    }
+    jo.put("module_to", moduleTo);
+    tenantOp(tenantId, jo);
+  }
+
+  private static void removeTenant(String tenantId) {
+
```

Comment:
For methods where the signature does not span multiple lines, I don't think the whitespace gaps improves readability, please remove it.

Reply:
Done